spec/whois/parsers/base_spec.rb

```diff
@@ -71,16 +71,21 @@
   end
 
   describe "#content_for_scanner" do
+    it "returns the part body with line feed normalized" do
+      instance = described_class.new(Whois::Record::Part.new(:body => "This is\r\nthe response.\r\n", :host => "whois.example.test"))
+      expect(instance.send(:content_for_scanner)).to eq("This is\nthe response.\n")
+    end
+
     it "returns the part body with line feed normalized" do
       instance = described_class.new(Whois::Record::Part.new(:body => "This is\r\nthe response.", :host => "whois.example.test"))
-      expect(instance.send(:content_for_scanner)).to eq("This is\nthe response.")
+      expect(instance.send(:content_for_scanner)).to eq("This is\nthe response.\n")
```

Comment:
So I understand that it's better for the scanner to ingest whole lines (including linebreak) instead of just the string on it, thus rendering the line empty but still present, right?

Reply:
Yes